assets/training/finetune_acft_image/components/finetune/hf_classification/spec.yaml

```diff
@@ -0,0 +1,352 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+version: 0.0.6
+name: transformers_image_classification_finetune
+display_name: Image Classification HuggingFace Transformers Model Finetune
+description: Component to finetune HuggingFace transformers models for image classification.
+
+is_deterministic: false
+
+environment: azureml://registries/azureml/environments/acft-transformers-image-gpu/versions/1
```

Comment:
nit: Here, we have env version 1, but env has version set to auto.

Reply:
Here, we need to have it like this.